contracts/rewards/interfaces/IAutoCompoundingRewards.sol

```diff
@@ -44,21 +45,35 @@ interface IAutoCompoundingRewards is IUpgradeable {
     function isProgramActive(Token pool) external view returns (bool);
 
     /**
-     * @dev creates a program for a pool
+     * @dev creates a flat program for a pool
```

Comment:
thoughts re.  changing to `creates a rewards program for a pool, with flat distribution`?

Reply:
Done:
-  creates a rewards program with flat distribution for a given pool
-  creates a rewards program with exponential-decay distribution for a given pool